package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbp/nexus-sdk",
-  "version": "1.0.0-beta.21",
+  "version": "1.0.0-beta.22",
```

Comment:
It's fine just now but don't change the package version in a PR like this. We change it when we want to do a release, which might include several PRs